inc/genrenator.php

```diff
@@ -49,9 +49,7 @@ function concat_fragments( $pattern ) {
 			$vector     = new $class_name();
 
 			$shards[ $i ] = str_replace( $fragment, $vector->texturize(), $fragment );
-			break;
 		}
-		$shards [ $i ] = $piece;
```

Comment:
@jazzsequence I think we needed line 54, but as the `else`...it should pass along array value that are not parsable fragments. `/` is the only one that comes to mind in pattern `'%prefix%#%genre%#/#%genre%#%suffix%',`...but any future characters added that end up as an array value need to remain intact for the eventual implosion.

Will you double-check me on that before I merge please?